tests/lib/rules/jsx-indent.js

```diff
@@ -1130,22 +1135,9 @@ const Component = () => (
       '  );',
       '}'
     ].join('\n'),
-    // The detection logic only thinks <App> is indented wrong, not the other
-    // two lines following. I *think* because it incorrectly uses <App>'s indention
-    // as the baseline for the next two, instead of the realizing the entire three
-    // lines are wrong together. See #608
-    /* output: [
-      'function App() {',
-      '  return (',
-      '    <App>',
-      '      <Foo />',
-      '    </App>',
-      '  );',
-      '}'
-    ].join('\n'), */
     options: [2],
     errors: [{message: 'Expected indentation of 4 space characters but found 0.'}]
-  }, {
```

Comment:
This test can't pass because `jsx-indent` is applying mistakenly indent.
I ignored the test.  I could not understand How to fix it 😭